build/Targets/VSL.Versions.targets

```diff
@@ -82,6 +82,16 @@
     </Otherwise>
   </Choose>
 
+  <PropertyGroup>
+    <GitHeadSha>Not found</GitHeadSha>
```

Comment:
Visual Studio Online defines an environment variable BUILD_SOURCEVERSION which is already the SHA1 of the thing that was checked out. Why not just use that?